test/schema.json

```diff
@@ -0,0 +1,204 @@
+{
+  "title": "Karst API Response",
+  "description": "Data payload for rendering the Karst App",
+  "type": "object",
```

Comment:
What does it mean that this is an `object`? Does that mean a JS object (`{ . . . }`) with an unspecified shape?

Reply:
Or is this literally always set to the string `"object"`?

Reply:
Oh, I think I understand now, this JSON is describing an object that looks like this:
```
{
  knacks: { . . . },
  pages: { . . . },
  tree:  { . . . },
}
```

Reply:
Yeah! https://json-schema.org/learn/ This was my first time using it but it seemed like a reasonable way to approach documenting this, and it was very easy to then include the validation of it as a test.